components/bpmn/bpmn-explorer-web/src/web/js/actions.js

```diff
@@ -1005,3 +1004,446 @@ function generateReport(){
     h3.hidden= true;
 
 }
+
+//Gets the details of the user-tasks in a completed process instance
+function getUserTasksOfCompletedProcessInstances(id){
```

Comment:
No its not : https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L1008